lib/go-atscfg/parentdotconfig.go

```diff
@@ -1694,9 +1694,6 @@ func getOriginServersAndProfileCaches(
 		} else if cgSv.TypeID == nil {
 			warnings = append(warnings, "getting origin servers: got server with nil TypeID, skipping!")
 			continue
-		} else if cgSv.ProfileID == nil {
-			warnings = append(warnings, "getting origin servers: got server with nil ProfileID, skipping!")
-			continue
```

Comment:
Shouldn't this be replaced with a check for `len(cgSv.ProfileNames) == 0`?

Reply:
yup